josm.openstreetmap.de.yaml

```
@@ -0,0 +1,72 @@
+app-id: josm.openstreetmap.de
+branch: stable
+tags:
+  - osm
+runtime: org.freedesktop.Platform
+runtime-version: 18.08
```

This is a string. Should be like this instead
```suggestion
runtime-version: '18.08'
```